contracts/TokenConversionLib.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity 0.4.23;
+
+import "./SafeMath.sol";
+
+library TokenConversionLib {
+    using SafeMath for uint256;
+
+    /**
```

Comment:
Suggest making the documentation of this function less specific, because the question of converted amount need not be limited to the staking context. It's just: given an amount of ST and the factors of a conversion rate, this function returns what the UT into which that amount of ST converts.

Please make "utility" in "Calculate Utility token..." lowercase and please update this documentation (and the documentation across this file) to be consistent with the new style guide (i.e., start with an uppercase letter, except in the case of proper nouns that are lowercased (e.g., variable names), and end with a period).

Reply:
1. Done, Removed stake and unstake terminology. Also changing ST to VT(value token)

2. Done 👍